cpp/src/slice2java/Gen.cpp

```diff
@@ -2262,14 +2253,14 @@ Slice::Gen::PackageVisitor::visitModuleStart(const ModulePtr& p)
 
         close();
     }
-    return false;
+    return true;
```

Comment:
This causes a change in the generated code. Before, when we always returned `false`, we would only ever hit top-level modules, but now we're hitting all modules.

The `if` check at the top of this function should be changed to:
```
if (!prefix.empty() && dynamic_pointer_cast<Unit>(p->container()))
```
this way we only generate `_Marker` interfaces for top-level module, like we were previously.

Reply:
Fixed.